src/main/java/gov/usds/case_issues/controllers/HitlistApiController.java

```diff
@@ -55,6 +61,37 @@
 		return _listService.getCases(caseManagementSystemTag, caseTypeTag, query);
 	}
 
+	@GetMapping
+	public List<CaseSummary> getCases(
+			@PathVariable String caseManagementSystemTag,
+			@PathVariable String caseTypeTag,
+			@RequestParam(required=true) CaseSnoozeFilter mainFilter,
+			@RequestParam(required=false) @DateTimeFormat(iso=ISO.DATE_TIME) ZonedDateTime caseCreationRangeBegin,
+			@RequestParam(required=false) @DateTimeFormat(iso=ISO.DATE_TIME) ZonedDateTime caseCreationRangeEnd,
```

Comment:
An alternative approach here might be to stop specifying `required` as a `@RequestParam` attribute, and instead use `Optional<ZonedDateTime>` as the type, so we can use a nice `isPresent()` instead of an ugly `!= null`.